lib/main.dart

```diff
@@ -29,6 +29,19 @@ void main(List<String> args) async {
   var path = '';
   var projectName = '';
   var designType = 'sketch';
+  var configurationPath;
```

Comment:
Hi @asaravindkumar , this line 32 causes an error with line 45 since configurationPath is declared again

Reply:
@ivan-015  apologies. I removed the variable creation and committed it

Reply:
Thank you!